src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: "
+                + "Circle"
+                + ","
+                + " area: "
+                + getArea()
+                + " sq. units,"
+                + " radius: "
+                + radius
+                + " units,"
+                + " color: "
+                + getColor());
```

Comment:
To many new lines - at some point it makes code less readable. How about a line per property? you can use a line separator (`\n`) to print output in multiple lines:
```java
System.out.println("Figure: circle,\n"
  "area: " + getArea() + ",\n"
  ...
```

Reply:
This comment applies to every `draw()` implementation.